src/main/resources/assets/botania/lang/en_US.lang

```diff
@@ -155,13 +155,20 @@ botaniamisc.altarRefill1=add back the last recipe
 botaniamisc.santaweaveInfo=Manaweave, changed up for the &cs&fe&ca&fs&co&fn
 botaniamisc.enterToView=^ Press ENTER to view ^
 botaniamisc.keyCategory=Botania
-botaniamisc.toggleLoki=Toggle Ring of Loki
-botaniamisc.toggleLokiBreaking=Toggle Ring of Loki Breaking
+
+#LOKI
+botaniamisc.toggleLoki=Ring of Loki Switch Key
 botaniamisc.ringOfLokiClear=Clear Ring of Loki Selection
 botaniamisc.lokiOn=On
 botaniamisc.lokiOff=Off
 botaniamisc.breaking=Breaking
 botaniamisc.lokiClear=Successfully cleared all selection
+botaniamisc.lokiDescription=Builder's wand but §c§oC§6§oo§e§oo§a§ol§9§oe§1§or
+botaniamisc.lokiDescription2=Can Place/Interact/Break with multiple blocks at once!
+botaniamisc.lokiToggleDescription=Mode Switch Key to turn ring
+botaniamisc.lokiBreakingDescription=Mode Switch Key + Shift to toggle breaking
+botaniamisc.lokiCurrent=Current
+botaniamisc.lokiState=Ring
```

Comment:
I didn't see Russian translations for these yet, did you forget them or was that on purpose?

Reply:
on purpose, I might do it later, I plan to give ring of loki tooltip more love soon so I will make translation when tooltip will be closer to final